_posts/2021-07-31-recently_soon1.md

```diff
@@ -0,0 +1,61 @@
+---
+author: nycticorax@opensuse.org & karatekhd@opensuse.org
```

Comment:
We prefer real names/nicknames here, not emails

Reply:
Okay like that? can you merge?

Reply:
Also perhaps if this is a blocker you might consider adding it to the README.md.